src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/kadas?serverTimezone=UTC
 spring.datasource.username=root
-spring.datasource.password=1234
+spring.datasource.password=${MYSQLDB_ROOT_PASSWORD}
```

Comment:
no need in this change, mentor is right, for .env vars to be resolved in properties 
spring.config.import=optional:file:.env[.properties]